### ISSUE TITLE
Unable to install VS Code plugin

### ISSUE THREAD

**Issue by willbowdi** [idolize/logitech-craft-vscode#3]
Hello,

System is running x64 Windows 10 with Visual Studio Code 1.32.2 & Node v10.15.1.

I have followed the README_DEV instructions and have successfully managed to install the Logitech Options plugin, however, I can't get VS Code to recognise the extension.

I've copied the full logitech-craft-vscode folder into the extensions folder (%USERPROFILE%\.vscode\extensions) as per the instructions in the VS Code documentation (https://vscode-docs.readthedocs.io/en/stable/extensions/install-extension/).

After running npm install while in the folder directory (%USERPROFILE%\.vscode\extensions\logitech-craft-vscode) and reloading the VS Code window I can't see the extension in the extensions list?

I've tried to add breakpoints and reload the window but nothing even shows in the debug console.

Kind regards,

Will

**Comment by idolize** [comment]
Hey @willbowdi I've published the VS Code extension to the marketplace now! https://marketplace.visualstudio.com/items?itemName=idolize.logitech-craft-vscode

Does this one work for you? Also, I'm working with Logitech to try to get the Options plugin officially signed as well.

**Comment by idolize** [comment]
Looks like I'm also having some issues on Windows - specifically getting Logitech Options to show the plugin in the list. I'm thinking maybe it's some issue with the manifest / defaults file...

**Comment by willbowdi** [comment]
Hi David, 

Thanks yes I can now download the plugin, however, I can't see anything in the extention output that it's picking up the commands from the keyboard.

For the windows plugin I had to manually create the following folder: ProgramData\Logishrd\**LogiOptionsPlugins** rather than ProgramData\Logishrd\LogiOptions\Plugins

**Comment by idolize** [comment]
@willbowdi Thanks for the tip about "LogiOptionsPlugins"! Are you sure you didn't add VSCode from the "Other applications" list? It's important that you select it from "Applications with Craft profiles" for the extension to work.

![Capture2](https://user-images.githubusercontent.com/635005/56404038-4900bb80-6219-11e9-8e40-fda0ab0faf36.PNG)

Anyway, I was finally able to get it working -- I just had to add a `"win_maximum_supported_version"` key/value to the manifest defaults file!

![Capture3](https://user-images.githubusercontent.com/635005/56404042-4d2cd900-6219-11e9-9851-47ee2fa8bfc4.PNG)

I just pushed an update to the repo (and the README) -- give it a shot now and let me know if it works!
